### ISSUE TITLE
Key modifiers are ignored (if they aren't valid)

### ISSUE THREAD

**Issue by Drarok** [kasper/phoenix#238]
- Version: 2.6.2 (71)
- macOS: 10.14.4 (18E226)

Phoenix binds to a key press, ignoring the modifiers I pass, meaning pressing the key on its own (such as when trying to type this message) triggers my callback.

I'm using the following code:
```js
Key.on('p', ['command', 'control'], function () {
  Phoenix.notify('Hello there!');
});
```

Edit:
This code also behaves the same as above, binding to just the `p` alone:
```js
let keyHandler = new Key('p', ['command', 'control'], function () {
  Phoenix.notify('Hello there!');
});
```

---

I've worked it out at last, but there's nothing in `Console.app` to tell me what's going wrong, and [the docs] are unclear, stating:

> Modifiers: command (cmd), option (alt), control (ctrl) and shift (case insensitive)

I read this to mean both `command` and `cmd` are valid, when it appears that _only_ `cmd` is valid. I would expect invalid modifiers to throw an error, or for Phoenix to accept both `command` and `cmd` as modifiers.

[the docs]: https://github.com/kasper/phoenix/blob/master/docs/API.md#1-keys

**Comment by kasper** [comment]
@Drarok `command`, `option` and `control` are implemented in the unreleased 2.6.3 (master), which I really need to get out. 😄

**Comment by kasper** [comment]
Fixed in 2.6.3.
